app/controllers/preview_controller.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+class PreviewController < ApplicationController
+  def show
+    keyword = Keyword.find(show_params[:id])
+
+    render partial: 'show', locals: {
```

Comment:
We directly don't render the partial from the controller, we render always partial view from the main view file if needed.
So I suggest render the main view file by renaming it `show.html` instead of `_show.html`
```suggestion
    render locals: {
```

Reply:
Interesting. I did not know the `layout: false` option and that is the reason why I called for a partial.
It's now fixed ;-)